src/bridge/IrcBridge.ts

```diff
@@ -522,16 +519,17 @@ export class IrcBridge {
         }
 
         // SUCCESS
-        this.bridge.getRequestFactory().addDefaultResolveCallback((req, res: string) => {
-            if (res === BridgeRequest.ERR_VIRTUAL_USER) {
+        this.bridge.getRequestFactory().addDefaultResolveCallback((req, _res) => {
+            const res = _res as BridgeRequestErr|null;
+            if (res === BridgeRequestErr.ERR_VIRTUAL_USER) {
```

Comment:
We use `BridgeRequestErr` instead of `string` now to ensure more type correctness.